packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "applications": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst6v8jmmTIJdv6ms0g4/app",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+
+## Sign On Rule Object
+
+| Property    | Type                                                                | Description                                                                           |
+|-------------|---------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| name        | String                                                              | Human-readable name for the Rule, configurable during creation or updating.           |
+| id          | String                                                              | Unique identifier for this Rule (read-only).                                          |
+| type        | String                                                              | Type of the Rule. For Identifier Match Rule Objects, this needs to be `Okta:SignOn`.  |
+| priority    | Integer                                                             | Used to determine which Rules take precedence.                                        |
+| conditions  | Array                                                               | UserType, User, Group and various Device conditions are supported for this rule type. |
+| action      | String                                                              | Either `ALLOW` or `DENY`. Controls whether the user is allowed to proceed.            |
+| requirement | [Sign On Rule Requirement Object](#sign-on-rule-requirement-object) | Specifies credentials to prompt user for.                                             |
+| status      | String                                                              | `ACTIVE`  or  `INACTIVE`.                                                             |
+| default     | Boolean                                                             | `true` for the first instance of this Rule, which gets created by default.            |
+
+#### Sign On Rule Requirement Object
+
+| Property                  | Type   | Description                                                                                                                                         |
+|---------------------------|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| `verificationMethod.type` | String | `CHAIN` or `ANY_FACTOR` to determine whether users should be prompted for a sequence of credentials or allowed to choose one credential from a set. |
```

Comment:
Just for curiosity
- how will the `requirement` looks like when the type is `ANY_FACTOR`?
- would that same as `type: CHAIN` with list of factors in `criteria` but no `next`? (@larsjohansen-okta @iancorcoran-okta )

Reply:
If type is `ANY_FACTOR` there will be no other properties in the `requirement` than `type`, since a chain wouldn't make sense to specify in that case.

Reply:
Sign On policy does not support `ANY_FACTOR` verification method, only `CHAIN`

Reply:
```
public void validate(Requirement requirement, Errors errors) {
        SignOnRequirement signOnRequirement = (SignOnRequirement) requirement;

        VerificationMethod verificationMethod = signOnRequirement.getVerificationMethod();

        // Must have a chain
        if (verificationMethod == null) {
            ErrorsHelper.rejectValue(errors, VERIFICATION_METHOD, PolicyErrorCode.INVALID_REQUIREMENT_VERIFICATION_METHOD.getErrorCode());
            return;
        }
        if (verificationMethod.getType() != VerificationMethodType.CHAIN) {
            ErrorsHelper.rejectValue(errors, VERIFICATION_METHOD_TYPE, PolicyErrorCode.INVALID_REQUIREMENT_VERIFICATION_METHOD_TYPE.getErrorCode());
            return;
        }

        FactorChainMethod factorChain = (FactorChainMethod) verificationMethod;
        if (factorChain.getChains() == null) {
            ErrorsHelper.rejectValue(errors, VERIFICATION_METHOD_CHAINS, PolicyErrorCode.INVALID_REQUIREMENT_EMPTY_VERIFICATION_METHOD_CHAINS.getErrorCode());
            return;
        }

        validateAllFactorChains(factorChain.getChains(), errors);
    }
```

Reply:
Yes, ugh, I mixed this up with the adoption feature's verification method. You're right. The only option is `CHAIN`.

Reply:
Changed to:

| `verificationMethod.type` | String | Currently, the only allowed value is `CHAIN`, which specifies that users should be prompted for a sequence of credentials. |